src/argilla/client/client.py

```diff
@@ -263,10 +241,12 @@ def log(
         workspace: Optional[str] = None,
         tags: Optional[Dict[str, str]] = None,
         metadata: Optional[Dict[str, Any]] = None,
-        batch_size: int = 500,
+        batch_size: int = 100,
         verbose: bool = True,
         background: bool = False,
         chunk_size: Optional[int] = None,
+        num_threads: int = 0,
+        max_retries: int = 3,
```

Comment:
```suggestion
        num_threads: int = 0,
        max_retries: int = 3,
        chunk_size: Optional[int] = None,
```
Let's keep deprecated parameters at the bottom.